evergreen/evergreen.yml

```diff
@@ -417,15 +417,16 @@ functions:
           ${PREPARE_SHELL}
           OCSP_TLS_SHOULD_SUCCEED="${OCSP_TLS_SHOULD_SUCCEED}" \
             OCSP_ALGORITHM=${OCSP_ALGORITHM} \
-            evergreen/add-certs.sh
+            SSL="ssl" \
```

Comment:
We should never be testing OCSP with `SSL="nossl"` so I've hardcoded `SSL="ssl"`